src/index.css

```
@@ -1,3 +1,19 @@
+ * This @import used to live inside HomePage.jsx's <style> block, which meant it
+ * only reached a page that mounted HomePage. /wheel and /stats are React.lazy
+ * standalone routes that never do, so the whole wheel surface — a surface whose
+ * DESIGN.md section runs to several thousand words about its material — was
+ * rendering in Arial, the platform sans. Nothing announced it, because every
+ * component states a size and a weight and none of them states a family.
+ *
+ * Moving it here changes nothing that does not ask for Barlow by name: a font
+ * that is loaded but unreferenced costs one cached request and no pixels. The
+ * URL is byte-identical to HomePage's so the two share one cache entry.
+ */
+@import url('https://fonts.googleapis.com/css2?family=Barlow:wght@400;500;600&family=Barlow+Condensed:wght@600;700;800;900&display=swap');
```

Stylelint (17.14.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Barlow:wght@400;500;600&family=Barlow+Condensed:wght@600;700;800;900&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Barlow:wght@400;500;600&family=Barlow+Condensed:wght@600;700;800;900&display=swap'" (import-notation)

(import-notation)